src/lib/duration/humanize.js

```diff
@@ -79,33 +79,27 @@ export function getSetRelativeTimeThreshold(threshold, limit) {
     return true;
```

Comment:
After your changes I got "export 'duration' (imported as 'moment') was not found in 'moment'"